lib/vanagon/platform/dsl.rb

```diff
@@ -221,13 +221,53 @@ def docker_image(name)
         @platform.docker_image = name
       end
 
+      def aws_ami(ami)
+        @platform.aws_ami = ami
+      end
+
+      def aws_user_data(userdata)
+        @platform.aws_user_data = userdata
+      end
+
+      def aws_region(region = 'us-east-1')
+        @platform.aws_region = region
+      end
+
+      def aws_shutdown_behavior(shutdown_behavior = 'terminate')
+        @platform.aws_shutdown_behavior = shutdown_behavior
+      end
+
+      def aws_key_name(key_name = 'vanagon')
+        @platform.aws_key_name = key_name
+      end
+
+      def aws_instance_type(instance_type = 't1.micro')
+        @platform.aws_instance_type = instance_type
+      end
+
+      def aws_vpc_id(vpc_id)
+        @platform.aws_vpc_id = vpc_id
+      end
+
+      def aws_subnet_id(subnet_id)
+        @platform.aws_subnet_id = subnet_id
+      end
+
       # Set the port for ssh to use if it's not 22
       #
       # @param port [Integer] port number for ssh
       def ssh_port(port = 22)
         @platform.ssh_port = port
       end
 
+      def target_user(user = "root")
```

Comment:
Yeah. We try to keep yard updated to that when we publish the gem, the API doc gets auto-updated.